omise/setting.php

```diff
@@ -0,0 +1,86 @@
+<?php
```

Comment:
@guzzilar Done. 7797ced697e0a40d00ea811e34afa7d5335f23e4

A function, getPublicKey(), has been added the DocBlock in the [next pull request](https://github.com/omise/omise-prestashop/pull/11/files#diff-726b73960d58daf9fe48ddd631df8a5cR21).